src/js/theme.ts

```diff
@@ -38,14 +38,16 @@ import initResponsiveToggler from './responsive-toggler';
 import initQuantityInput from './qty-input';
 import initQuickview from './quickview';
 import initCart from './pages/cart';
-import initCheckout from './checkout';
+import initCheckout from './pages/checkout';
 import useToast from './components/useToast';
 import useProgressRing from './components/useProgressRing';
 import './modules/blockcart';
 import initProductBehavior from './product';
 import './mobile-menu';
 import './modules/ps_searchbar';
 import './modules/facetedsearch';
+import './modules/ps_languageselector';
+import './modules/ps_currencyselector';
```

Comment:
@NeOMakinG, is there any chance to force webpack to export these files as separated JS files to proper directories? I'm wondering what if one decides to disable these modules. Will we end up having a bunch of not needed JS?

Reply:
@kpodemski maybe someone wants to disable facetedsearch, mobile-menu, ...
I think having just 10 lines of JS code in bundle doesn't impact the performance.

Reply:
This is something we need to think about, It should be possible

Reply:
Yes, we need to revert to this. In 1.6, due to separation of css files, only the global.css + bundles related to the controller were loaded. In 1.7, at least modules stayed “modular”.

We cannot make it even worse and make everything into one huge bundle loading everything even if it's not needed. 😬

CCC system is ready for this, it caches unique packs of assets.

Reply:
@jolelievre you got it, we changed the module markup, so we need to handle things with our own markup now